components/kyma-environment-broker/internal/process/deprovisioning/initialisation.go

```diff
@@ -23,6 +23,8 @@ import (
 const (
 	// the time after which the operation is marked as expired
 	CheckStatusTimeout = 5 * time.Hour
+
+	operationTimeout = time.Hour * 24
```

Comment:
What do you think about having this parameter configurable via environmental variables?